packages/hash/frontend/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "next/babel",
+      {
+        "preset-env": {
+          "targets": "defaults"
```

Comment:
I know we added this somewhere else but I forgot what it's for, can you explain?

Reply:
This is a mistake, removing

Reply:
https://github.com/hashintel/hash/commit/f036c729cbe720157dfb9a332c78a25411eccb7f